src/fides/api/models/privacy_experience.py

```diff
@@ -215,6 +217,9 @@ class PrivacyExperience(PrivacyExperienceBase, Base):
     # related to experiences.
     privacy_notices: List[PrivacyNotice] = []
 
+    # Attribute that is cached on the PrivacyExperience object by "get_should_show_banner", calculated at runtime
+    show_banner: bool
```

Comment:
somewhat related to my comment below about considering whether we still need this db table/model - it just struck me as a bit odd that this property is on the db model. i can see why you've done it, and i'm not saying we should change it now...just some food for thought.

Reply:
I do like how the Experience has the all the details for a single region.  `show_banner` really is a property of a single region.  It's looking at notices for that region, and then its ExperienceConfig via the FK to see if this specific region should surface a banner!